SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -462,20 +473,20 @@ private static void ShowUsage()
         {
             Console.WriteLine("Usage:");
             Console.WriteLine("DeploymentLauncher create <project-name> <assembly-name> <main-deployment-name> <main-deployment-json> <main-deployment-snapshot> <main-deployment-region> [<sim-deployment-name> <sim-deployment-json> <sim-deployment-region> <num-sim-players>]");
-            Console.WriteLine($"  Starts a cloud deployment, with optionally a simulated player deployment. The deployments can be started in different regions ('EU', 'US' and 'AP').");
-            Console.WriteLine("DeploymentLauncher stop <project-name> [deployment-id]");
+            Console.WriteLine($"  Starts a cloud deployment, with optionally a simulated player deployment. The deployments can be started in different regions ('EU', 'US' , 'AP' and 'CN').");
+            Console.WriteLine("DeploymentLauncher stop <project-name> <main-deployment-region> [deployment-id]");
```

Comment:
Can this argument for `stop` and `list` commands be optional? The API endpoint would default to null if the argument isn't present or if it's a region different from `CN`.

Reply:
So this one is a little difficult to do because of the argument ordering, the cmd line is a set of ordered arguments which are unnamed so making things optional is a bit difficult (the sim-deployment stuff kind of took the only optional space) - it might be worth me adding a task to fix these in the dev branch

Reply:
Ah, you're right. It would have been easier if it was the only optional argument, but in this case since `[deployment-id]` is optional, making it so you could provide the deployment id but not the region would be tricky.
Adding a task to look at this later sounds good to me.